lib/tokenizers.js

```diff
@@ -239,13 +234,12 @@ export async function _createTokenizer() {
     secret: base64url.encode(secret),
     state: 'pending'
   };
-  let record = {
+  const record = {
     meta,
     tokenizer
   };
   try {
-    const result = await collection.insertOne(record, database.writeOptions);
-    record = result.ops[0];
```

Comment:
`record` assigned here on L248 is never used below.  Therefore `result` is not needed, and `record` can be a `const`.